pages/register.tsx

```diff
@@ -0,0 +1,42 @@
+// import { csrfToken, signIn } from 'next-auth/client';
```

Comment:
This whole page will be replaced by @Sonjajo 's PR, this is just a placeholder for now to test

Reply:
sonja's PR current PR doesn't cover user sign up and sign in

Reply:
this should be at `users/signup`